src/components/ui/Pool/ZapOutSection.tsx

```
@@ -0,0 +1,157 @@
+          options={zapTokenOptions}
+          className="order-first sm:order-none sm:col-span-4 col-span-12 max-w-[10rem]"
+          onChange={handleZapTokenChange}
+          disabled={!poolTokens.length}
+        />
+            </div>
+          </div>
+          <Button
+            onClick={
+              isConnected
+                ? () => onZapOut(zapAmount, zapTokenAddress as Hex)
+                : () => openConnectModal && openConnectModal()
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)